app/assets/stylesheets/darkswarm/embedded_shopfront.css.scss

```diff
@@ -51,6 +51,15 @@ nav.top-bar ul.right li.powered-by {
   }
 }
 
+.powered-by-embedded {
+   opacity: 0.6;
+   font-family: "Oswald", sans-serif;
```

Comment:
I haven't looked at any scss file in OFN so far, so I wonder if we have a var for the font-family already.